src/codegen/generate.ts

```diff
@@ -880,7 +963,8 @@ export default class ApiGenerator {
     Object.assign(stub, {
       statements: cg.appendNodes(
         stub.statements,
-        ...[...this.aliases, ...functions]
+        ...[...this.aliases, ...functions],
+        ...this.enumAliases
       ),
     });
 
```

Comment:
Just a bit of context on why I did this way. I had to treat Enums as references in order to allow us to reuse already declared, and create new ones. It needed its own "hashmap" cause otherwise, the names could conflict with the references.

Reply:
Hey, @Xiphe any updates on this?